dbt/include/duckdb/macros/adapters.sql

```diff
@@ -157,7 +157,7 @@ def materialize(df, con):
         END as type
     from system.information_schema.tables
     where table_schema = '{{ schema_relation.schema }}'
```

Comment:
macro question (so to speak): do we have the same issue for schemas w/respect to cases and what not? Do we need to do the same casing stuff for schemas as well?

Reply:
yeah why not. I'd be OK with extending this to `generate_schema_name` as well